src/node_zlib.cc

```diff
@@ -517,12 +523,28 @@ class ZCtx : public AsyncWrap {
         break;
       case INFLATE:
       case GUNZIP:
-      case INFLATERAW:
       case UNZIP:
         ctx->err_ = inflateInit2(&ctx->strm_, ctx->windowBits_);
         ctx->env()->isolate()
             ->AdjustAmountOfExternalAllocatedMemory(kInflateContextSize);
         break;
+      case INFLATERAW:
+        ctx->err_ = inflateInit2(&ctx->strm_, ctx->windowBits_);
+        ctx->env()->isolate()
+            ->AdjustAmountOfExternalAllocatedMemory(kInflateContextSize);
+        if (ctx->dictionary_ != nullptr) {
+          ctx->err_ = inflateSetDictionary(&ctx->strm_,
+                                           ctx->dictionary_,
+                                           ctx->dictionary_len_);
+          if (ctx->err_ != Z_OK) {
+            // We can't detect mismatching dictionaries in raw mode since we
+            // don't have the dictid from the wrapper, thus the only failures
```

Comment:
Again, just a suggestion, but using personal pronouns in the code/comments is generally discouraged around here.